src/platform/user/selectors.js

```diff
@@ -23,27 +20,15 @@ export const isMultifactorEnabled = state => selectProfile(state).multifactor;
 export const selectAvailableServices = state => selectProfile(state)?.services;
 export const selectPatientFacilities = state =>
   selectProfile(state)?.facilities?.map(({ facilityId, isCerner }) => {
-    // TODO: The work in this selector to override the `isCerner` values will be
-    // removed after the override logic gets moved to vets-api. ie, we will be
-    // able to trust the `isCerner` flags that come directly from vets-api.
-
     // Derive if the user belongs to a Cerner facility in the FE maintained list.
     const hasCernerFacilityID = CERNER_FACILITY_IDS.includes(facilityId);
 
-    // Derive if the feature toggle is on.
-    // TODO: can this feature toggle check be removed since it should always be true now?
-    const showNewScheduleViewAppointmentsPage =
-      state?.featureToggles?.[
-        featureFlagNames.showNewScheduleViewAppointmentsPage
-      ];
-
-    // Derive if they are a 200CRNR Cerner patient.
-    const isCernerPatient = selectProfile(state)?.isCernerPatient;
-
     // Derive if we should consider it a Cerner facility.
+    const isFlipperDisabled =
+      state?.featureToggles?.[`cerner_override_${facilityId}`] === false;
+    const isFlipperEnabled = !isFlipperDisabled;
     const passesCernerChecks =
-      showNewScheduleViewAppointmentsPage &&
-      (isCerner || (isCernerPatient && hasCernerFacilityID));
+      isFlipperEnabled && (isCerner || hasCernerFacilityID);
```

Comment:
If `isFlipperEnabled` is changed as suggested above, I think we could make this: `isCerner || isFlipperEnabled` and stop using the `CERNER_FACILITY_IDS`. Seems like the `CERNER_FACILITY_IDS` and Flipper flags are sort of redundant/conflicting. We can use the Flipper flags and only the Flipper flags to force `isCerner` to true.

Reply:
We can make updates to this afterwards if that's ok, we need to get this in before the daily depoy today

Reply:
There's the case where if the feature toggle does not exist, we want to rely on MPI's `isCerner` or the `CERNER_FACILITY_IDS` to determine if the facility should be considered a Cerner facility

Reply:
We have that suggestion on `CERNER_FACILITY_IDS` listed as a potential action item for after this launch

Reply:
We are also considering never even looking at `isCerner` true and always deriving it from the Flipper flag, so that certain facilities are not launched prematurely in the event `isCerner` turns to true before it's ready (which is possibly happening with this latest facility)

Reply:
okay so I am missing a lot of the crap you are having to deal with here! :homer-disappear: